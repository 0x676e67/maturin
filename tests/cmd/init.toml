bin.name = "maturin"
args = "init --help"
