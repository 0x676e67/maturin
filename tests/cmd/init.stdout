maturin[EXE]-init 
Create a new cargo project in an existing directory

USAGE:
    maturin[EXE] init [OPTIONS] [PATH]

ARGS:
    <PATH>    Project path

OPTIONS:
        --name <NAME>            Set the resulting package name, defaults to the directory name
        --mixed                  Use mixed Rust/Python project layout
        --src                    Use Python first src layout for mixed Rust/Python project
    -b, --bindings <BINDINGS>    Which kind of bindings to use [possible values: pyo3, rust-cpython,
                                 cffi, bin]
    -h, --help                   Print help information
